generate_release.py

```diff
@@ -16,8 +16,8 @@
 sha256_hash = hashlib.sha256()
 checksum = "missing"
 shafiles = [
-    "ElevenClock.Installer.exe",
     "ElevenClock.exe",
+    "ElevenClock.Installer.exe",
```

Comment:
This will be does not works, if "ElevenClock.exe" will be exists, then checksum will be created only for him. See `break` on line 29.

Reply:
Aaaaa makes sense

Reply:
Reverted!